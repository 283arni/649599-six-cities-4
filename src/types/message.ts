export interface messageType {
  status: number;
  data: {
    error: string;
  };
}
