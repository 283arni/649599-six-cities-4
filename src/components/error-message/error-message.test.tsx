import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ErrorMessage from './error-message';
import {messageServer} from '../../mocks/test/message';

it(`check render ErrorMessage`, () => {

  const tree = renderer.create(
      <ErrorMessage
        messageServer={messageServer}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
