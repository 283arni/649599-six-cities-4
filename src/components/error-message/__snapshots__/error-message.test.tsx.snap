// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check render ErrorMessage 1`] = `
<div
  className="error"
  style={
    Object {
      "backgroundColor": "red",
      "border": "2px solid pink",
      "borderRadius": "3px",
      "color": "pink",
      "fontSize": "1.5rem",
      "left": "50%",
      "minWidth": "300px",
      "padding": "10px",
      "position": "fixed",
      "textAlign": "center",
      "top": "20px",
      "transform": "translateX(-50%)",
      "zIndex": 100,
    }
  }
>
  ok Status: 200
</div>
`;
